app/Filters/BankIntegrationFilters.php

```diff
@@ -93,6 +94,10 @@ public function status(string $filter = ''): Builder
     public function sort(string $sort): Builder
```

Comment:
We'll need to use a default here (and any other methods that are missing this).

sort(string $sort='')

Reply:
done